RealmSwift/Impl/ObjcBridgeable.swift

```diff
@@ -163,6 +163,27 @@ extension AnyRealmValue: BuiltInObjcBridgeable {
     }
 }
 
+
+extension RLMManagedDictionary: BuiltInObjcBridgeable {
```

Comment:
These are not valid implementations of BuiltInObjcBridgeable, and declaring these types as BuiltInObjcBridgeable doesn't really make sense at all.